migration/migrating_3_4/migrating-applications-with-cam-3-4.adoc

```diff
@@ -20,9 +24,15 @@ You can configure a migration plan to suit your needs by increasing the number o
 include::modules/migration-changing-migration-plan-limits.adoc[leveloffset=+2]
 include::modules/migration-excluding-resources.adoc[leveloffset=+2]
 
-include::modules/migration-adding-cluster-to-cam.adoc[leveloffset=+1]
-include::modules/migration-adding-replication-repository-to-cam.adoc[leveloffset=+1]
-include::modules/migration-creating-migration-plan-cam.adoc[leveloffset=+1]
-include::modules/migration-running-migration-plan-cam.adoc[leveloffset=+1]
+[role="_additional-resources"]
+== Additional resources
+
+* link:https://docs.openshift.com/container-platform/3.11/install_config/registry/securing_and_exposing_registry.html#exposing-the-registry[Exposing a secure registry manually on an {product-title} 3 cluster]
+* xref:../../registry/securing-exposing-registry.adoc#registry-exposing-secure-registry-manually_securing-exposing-registry[Exposing a secure registry manually on an {product-title} 4 cluster]
+* xref:../../migration/migrating_3_4/migrating-application-workloads-3-4.adoc#file-system-copy-method_migrating-3-4[{mtc-short} file system copy method]
```

Comment:
Are these additional resources linking to sections in this same assembly? Just not sure if that's necessary since it's in the same assembly?

Reply:
No, they're not. They are links to the conceptual material in another assembly. The assembly titles sound similar because I did not know at the time what direction the content would go. Once they were created, I couldn't change them.